x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/rank/textsimilarity/TextSimilarityRankTests.java

```diff
@@ -178,6 +186,38 @@ public void testRerankInferenceFailure() {
         );
     }
 
+    public void testLenientRerankInference() {
+        ElasticsearchAssertions.assertNoFailuresAndResponse(
+            // Execute search with text similarity reranking
+            client.prepareSearch()
+                .setRankBuilder(
+                    new TextSimilarityTestPlugin.ThrowingMockRequestActionBasedRankBuilder(
+                        100,
+                        "text",
+                        "my-rerank-model",
+                        "my query",
+                        null,
+                        true,
+                        AbstractRerankerIT.ThrowingRankBuilderType.THROWING_RANK_FEATURE_PHASE_COORDINATOR_CONTEXT.name()
+                    )
+                )
+                .setQuery(QueryBuilders.matchAllQuery()),
+            response -> {
+                // these will all have a score of 2 (default 1 + normalization)
```

Comment:
Should we still normalize if the reranker failed?

Reply:
Should we also still follow `minScore`, or does that only apply to reranked scores?

Reply:
The rankers normalization is just to prevent negative scores, not actually min/max normalization. So, anything that is providing final scores other than the ranker is likely to be on a different distribution scale than the one that the original `minScore` was calibrated with.

I would ignore it.